datamodel.customer-survey.xml

```diff
@@ -1955,21 +1957,16 @@ EOF
         </method>
         <method id="AppendLinkToContent">
           <comment>/**
-	 * @param boolean $bAutoAppendLink if false will always return the content, otherwise will append a quiz link if no url placeholder found in content
-	 * @param string $sContent
-	 * @param ?string $sQuizUrl
-	 *
-	 * @return string
-*/</comment>
+            * @param string $sContent
+            * @param string $sQuizUrl
+            *
+            * @return string If a placeholder is found in $sContent returns $sContent, otherwise $sContent with a quizz link appended
+            */</comment>
```

Comment:
Not really important, but now these comments will look weird in the generated php code (way too much indentation)